app/_src/guides/gateway-builtin.md

```diff
@@ -204,8 +204,9 @@ spec:
     name: demo-app_kuma-demo_svc_5000
   from:
     - targetRef:
-        kind: MeshService
-        name: edge-gateway 
+        kind: MeshSubset
```

Comment:
Makes me wonder whether we should support `MeshGateway` as `from`, it doesn't have the same problems as `MeshService`

Reply:
Yeah, but I think at this moment there is an odd behaviour with multiple selectors https://github.com/kumahq/kuma/issues/9185#issuecomment-1973363863